presto-ui/src/router/ClusterHUD.jsx

```diff
@@ -32,124 +32,136 @@ const SPARKLINE_PROPERTIES = {
     disableHiddenCheck: true,
 };
 
-export class ClusterHUD extends React.Component {
-    constructor(props) {
-        super(props);
-        this.state = {
-            runningQueries: [],
-            queuedQueries: [],
-            blockedQueries: [],
-            activeWorkers: [],
-            runningDrivers: [],
-            reservedMemory: [],
-            rowInputRate: [],
-            byteInputRate: [],
-            perWorkerCpuTimeRate: [],
-            clusterCount: [],
+export const ClusterHUD = () => {
+    const [state, setState] = useState({
+        runningQueries: [],
+        queuedQueries: [],
+        blockedQueries: [],
+        activeWorkers: [],
+        runningDrivers: [],
+        reservedMemory: [],
+        rowInputRate: [],
+        byteInputRate: [],
+        perWorkerCpuTimeRate: [],
+        clusterCount: [],
 
-            lastRender: null,
-            lastRefresh: null,
+        lastRender: null,
+        lastRefresh: null,
 
-            lastInputRows: null,
-            lastInputBytes: null,
-            lastCpuTime: null,
+        lastInputRows: null,
+        lastInputBytes: null,
+        lastCpuTime: null,
 
-            initialized: false,
-        };
+        initialized: false,
+    });
 
-        this.refreshLoop = this.refreshLoop.bind(this);
-    }
+    const timeoutId = useRef(null);
 
-    resetTimer() {
-        clearTimeout(this.timeoutId);
-        // stop refreshing when query finishes or fails
-        if (this.state.query === null || !this.state.ended) {
```

Comment:
Neither of these properties were in the `state` object to begin with, so removing check